express/css/custom.css

```diff
@@ -98,12 +92,15 @@ font-family: 'Questrial', sans-serif;
     display: block;
   }
   
-  /* Change color of dropdown links on hover */
-  .dropdown-content a:hover {background-color: #ddd}
-  
-  /* Show the dropdown menu (use JS to add this class to the .dropdown-content container when the user clicks on the dropdown button) */
-  .show {display:block;}
 
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {background-color: #ddd;}
+
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content {display: block;}
+
+/* Change the background color of the dropdown button when the dropdown content is shown */
```

Comment:
also would probably get rid of as many of these comments as possible, if you've named your classes well (which you have!) then it should be pretty obvious what is going on. Comments are messy and another thing to maintain